client/src/locales/en-US.json

```diff
@@ -203,7 +203,18 @@
             "tosLink": "https://my.parsec.cloud/assets/pdf/Parsec-CGVU-en.pdf",
             "and": "and ",
             "privacyPolicy": "Privacy Policy",
-            "privacyPolicyLink": "https://parsec.cloud/en/politique-de-confidentialite/"
+            "privacyPolicyLink": "https://parsec.cloud/en/politique-de-confidentialite/",
+            "update": {
+                "title": "New terms of service",
+                "subtitle": "We have updated our terms of service. Please read and accept them to continue.",
+                "tosButton": "Terms Of Service",
+                "url": "https://my.parsec.cloud/assets/pdf/Parsec-CGVU-fr.pdf",
```

Comment:
Placeholder ?

Reply:
Not sure what this URL text is about, but it should probably not point to Parsec CGVU.